tox.ini

```diff
@@ -1,12 +1,17 @@
 [tox]
 envlist =
     {py27,py36}-django111
+    {py36}-django{200, 210}
     flake8
     docs
 
 [testenv]
 deps =
     django111: Django>=1.11,<2.0
+    django200: Django>=2.0,<2.1
+    django210: Django>=2.1
```

Comment:
I prefer `Django>=2.1,<3.0`. So, it ensures django 3.x will not be installed when it is available.